packages/opentelemetry-instrumentation-xml-http-request/src/xhr.ts

```diff
@@ -45,6 +46,10 @@ import { AttributeNames } from './enums/AttributeNames';
 // safe enough
 const OBSERVER_WAIT_TIME_MS = 300;
 
+export interface XHRCustomAttributeFunction {
+  (span: api.Span, xhr: XMLHttpRequest): void;
+}
```

Comment:
Any reason this is an interface not a type?

```suggestion
export type XHRCustomAttributeFunction = (span: api.Span, xhr: XMLHttpRequest) => void;
```